### ISSUE TITLE
Cannot call hover while not dragging

### ISSUE THREAD

**Issue by faizanabidnaqvi** [Camberi/firecms#51]
Hi there!

I have a field that is of array type and stores string URL of uploaded images. The images are uploaded through the storageMeta config, which is set as follows:
```
imageUrls: {
            title: "Images",
            dataType: "array",
            of: {
                dataType: "string",
                config: {
                    storageMeta: {
                        mediaType: "image",
                        storagePath: "images",
                        acceptedFiles: ["image/png"],
                        storeUrl: true,
                    }
                }
            }
```

When I try to drag and drop images in this field, the app crashes with the following error:
<img width="1103" alt="Screenshot 2021-02-08 at 8 50 26 PM" src="https://user-images.githubusercontent.com/5749036/107267540-b36a9780-6a4f-11eb-91bf-ffa4f46d5ef7.png">

If I upload by clicking and selecting the files, then it works. Will be thankful if you could take a look. :)

**Comment by fgatti675** [comment]
Hi @faizanabidnaqvi can you check if this happens also in v0.25.0 ? 
Thanks!

**Comment by faizanabidnaqvi** [comment]
Hi @fgatti675 

Thanks for getting back to me. Just checked it. It happens in v0.25.0 as well.

**Comment by faizanabidnaqvi** [comment]
Seems to be fixed in 0.26.0 :)

**Comment by fgatti675** [comment]
I did not manage to reproduce this one, glad it works now!
